drivers/gles3/rasterizer_scene_gles3.h

```diff
@@ -328,6 +330,7 @@ class RasterizerSceneGLES3 : public RasterizerScene {
 		float atlas_clamp[4];
 		float local_matrix[16]; //up to here for spot and omni, rest is for directional
 		//notes: for ambientblend, use distance to edge to blend between already existing global environment
+		uint8_t padding[10];
```

Comment:
Reflection probes are not passed directly but as an array, so in practice the padding is not needed here